### PR TITLE
Add huggingface token for native llm

### PR DESCRIPTION
**Pull request by XinyaoWa**
## Description

Add huggingface token for native llm

## Issues

List the issue or RFC link this PR is working on. If there is no such link, please mark it as `n/a`.

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [ ] Others (enhancement, documentation, validation, etc.)

## Dependencies

List the newly introduced 3rd party dependency if exists.

## Tests

Describe the tests that you ran to verify your changes.

### CODE REVIEW

File: comps/llms/text-generation/native/langchain/utils.py [hunk 1]
```diff
@@ -42,6 +43,10 @@
 from transformers import AutoConfig, AutoModelForCausalLM, AutoTokenizer
 from transformers.utils import check_min_version
 
+HUGGINGFACEHUB_API_TOKEN = os.getenv("HUGGINGFACEHUB_API_TOKEN", "")
+if HUGGINGFACEHUB_API_TOKEN != "":
+    login(token=HUGGINGFACEHUB_API_TOK EN)
```
Comment (dcmiddle) [inline comps/llms/text-generation/native/langchain/utils.py:48]: Looks like an erroneous space?
```suggestion
    login(token=HUGGINGFACEHUB_API_TOKEN)
```

**Review by dcmiddle — COMMENTED** [review]
Consider rebasing/squashing your fixup to make cleaner commit history.

Your commit doesn't explain why we need to add this token. Does the comp not work without it?

**Review by lvliang-intel — APPROVED** [review]

**Comment by XinyaoWa** [comment]
> Consider rebasing/squashing your fixup to make cleaner commit history.
> 
> Your commit doesn't explain why we need to add this token. Does the comp not work without it?

hg token is needed if you want to access gated model(such as llama), otherwise no need.

**Comment by XinyaoWa** [comment]
> Consider rebasing/squashing your fixup to make cleaner commit history.
> 
> Your commit doesn't explain why we need to add this token. Does the comp not work without it?

Thanks for you suggestion about cleaning commit history, we will follow this for later PRs~

**Pull request merged**